### DISCUSSION TITLE
TokenSource field is never assigned

### DISCUSSION THREAD

**Discussion by VladislavAntonyuk** [CommunityToolkit/Maui#680] [General]
Seems like `tokenSource` field is used but never assigned, so its value is always `null`. Can we remove it?
https://github.com/CommunityToolkit/Maui/blob/a6d421d63def9eda1664f5f0c351bea912a8b6a8/src/CommunityToolkit.Maui/ImageSources/GravatarImageSource.shared.cs#L34

@GeorgeLeithead asking your expert opinion about it

**Comment by GeorgeLeithead** [comment]
@VladislavAntonyuk I will have a look over the weekend.

**Reply by VladislavAntonyuk** [reply]
@GeorgeLeithead Any chance you can take a look?

**Reply by GeorgeLeithead** [reply]
@VladislavAntonyuk I have looked and issued a PR: https://github.com/CommunityToolkit/Maui/pull/875

**Reply by GeorgeLeithead** [reply]
Was merged and released as part of 3.1 I believe.

**Comment by CliffAgius** [comment]
Closed and answered
